paddle/operators/linear_chain_crf_op.h

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
Will this header file also be used for the CRF decoding code? If not, it seems this header file can be removed in this PR.

Reply:
After our discussion. I will keep this header and move the function `Compute` into the header file and add GPU kernel function.

Reply:
Done.